pkg/registry/experimental/controller/etcd/etcd_test.go

```diff
@@ -79,31 +77,27 @@ var validScale = extensions.Scale{
 }
 
 func TestGet(t *testing.T) {
-	storage, fakeClient := newStorage(t)
+	storage, server, si := newStorage(t)
+	defer server.Terminate(t)
 
 	ctx := api.WithNamespace(api.NewContext(), "test")
 	key := etcdtest.AddPrefix("/controllers/test/foo")
-	if _, err := fakeClient.Set(key, runtime.EncodeOrDie(testapi.Default.Codec(), &validController), 0); err != nil {
+	if err := si.Set(ctx, key, &validController, nil, 0); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-
-	expect := &validScale
-	obj, err := storage.Get(ctx, "foo")
-	scale := obj.(*extensions.Scale)
+	_, err := storage.Get(ctx, "foo")
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if e, a := expect, scale; !api.Semantic.DeepEqual(e, a) {
```

Comment:
I added a replica check similar to what is done in the other test.  I've been trying to stay away from DeepDerivative tests where possible.  

Done.